yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala

```diff
@@ -99,7 +103,14 @@ class YarnClusterSuite extends SparkFunSuite with BeforeAndAfterAll with Matcher
     val logConfFile = new File(logConfDir, "log4j.properties")
     Files.write(LOG4J_CONF, logConfFile, UTF_8)
 
+    // Disable the disk utilization check to avoid the test hanging when people's disks are
+    // getting full.
+    val yarnConf = new YarnConfiguration()
+    yarnConf.set("yarn.nodemanager.disk-health-checker.max-disk-utilization-per-disk-percentage",
```

Comment:
Mostly just here. No other tests run an NM, and this just affects whether the NM will run containers or not. (In fact I could remove this now since I freed up some space locally, but it doesn't hurt to have it.)